bokehjs/src/lib/models/glyphs/webgl/buffer.ts

```diff
@@ -36,7 +36,7 @@ abstract class WrappedBuffer<ArrayType extends WrappedArrayType> {
   }
 
   get length(): number {
-    return this.array.length
+    return this.array != null ? this.array.length : 0
```

Comment:
If `array` can be `null` (or I suppose `undefined`, uninitialized), then this has to be encoded in the type of `array`, possibly `array?: ArrayType` or `array: ArrayType | null = null`. We currently don't enforce `strictPropertyInitialization` (due to `initialize()` methods), so enforcing this has to be done manually.

Reply:
It can be uninitialized. Up until now it has always been initialized before being used, but in this situation I need to check it before that.  I've opted for the `array?: ArrayType` approach.